docs/servo_power_test/src/test_runner.ts

```diff
@@ -83,6 +92,11 @@ export class IterationData {
   }
   public loadHistogramData() {
     this.histogramDataList = this.extractData(0);
+    this.powerSum = 0;
+    this.histogramDataList.forEach(powerData => {
+      this.powerSum += powerData;
+    });
+    this.powerAverage = this.powerSum / this.histogramDataList.length;
```

Comment:
this looks like the same logic as the constructor. could you refactor out this into another function?